doc/ci_admin_guide/Docker_registry_provisioning.md

```diff
@@ -0,0 +1,152 @@
+# Provisioning Docker registry
+
+## Preparing machine
+
+Spawn a VM using the CentOS / Ubuntu image (the rest of instruction will assume (CentOS 7),
```

Comment:
Disagreed. The general setup is OS specific, but the rest could be run on Ubuntu. Actually, usage of CentOS in CI infra is temporary, as most of machines (mgmt, jenkins, zuul) use ubuntu and we shouldn't make an exception here. I've used CentOS because the Ubuntu Server templated somehow vanished.